tests/test_pysugarcrm.py

```diff
@@ -59,3 +59,14 @@ def test_context_manager(self):
             self.assertEqual(api.is_closed, False)
 
         self.assertEqual(api.is_closed, True)
+
+    @responses.activate
+    def test_context_manager_without_access_token(self):
+        responses.add(
+            responses.POST,
+            'http://testserver.com/rest/v10/oauth2/token',
+            body=json.dumps({}),
+            status=200
+        )
+
+        self.assertRaises(ValueError, func=sugar_api('http://testserver.com/', "admin", "12345"))
```

Comment:
```suggestion
        with self.assertRaises(ValueError):
            sugar_api('http://testserver.com/', "admin", "12345")
```

I find something like this easier to read and understand, what do you think?

Reply:
looks better for me too.